packages/holocron-dev-server/src/utils/logs/messages.js

```diff
@@ -0,0 +1,270 @@
+    serverAddress,
+    getPublicVendorsUrl(getReportFilename(externalsBundleName))
+  );
+  log(
+    printWebpack(
+      [
+        `externals bundled: ${externals
+          .map((external) => green(external))
+          .join(', ')}`,
+        newLine(orange('Bundle Analyzer for Dll externals can be found at \n\n   %s')),
+      ].join('\n')
+    ),
+    yellow.bold(`"${externalsReportUrl}"\n`)
+  );
+}
+
+export function logModuleBundlerAnalyzerUrl(serverAddress) {
+  const analyzerHtml = joinUrlFragments(
+    serverAddress,
+    getPublicModulesUrl(getReportFilename(modulesBundleName))
+  );
+  log(
+    printWebpack(orange('Bundle Analyzer for local Holocron modules is available at \n\n   %s')),
+    yellow.bold(`"${analyzerHtml}"\n`)
+  );
+}
+
+// HTML
+
+export function logRenderedHolocronModules(modules) {
+  log(
+    `rendered HTML document using local modules: [ ${orange.bold(
+      modules.map(({ name }) => `"${name}"`).join(', ')
+    )} ]`
+  );
+}
+
+// Proxy
+
+export function logProxyRequestMatch(req) {
+  log(printProxy(`Matched remote for ${yellow('"%s"')}`), req.path);
+}
+
+export function logRemoteHasBeenLoadedCached(remoteUrl) {
+  log(printProxy(`Fetched and stored remote from ${yellow('"%s"')}`), remoteUrl);
+}
+
+// Mocks
+
+export function logMockAction({ fileName, moduleName, action }) {
+  log(
+    printMock(
+      `"${orange.bold(fileName)}" for module ${orange(`"${moduleName}"`)} has been ${action}`
+    )
+  );
+}
+
+export function logMockWatchReady() {
+  log(printMock('Watching "/mock/" directory for scenarios change'));
+}
+
+export function warnOnMockWatchError(e) {
+  warn(printMock(`Watch error: ${e}`));
+}
+
+export function logScenariosRegistered({ scenarios, serverAddress }) {
+  const spacer = '    ';
+  info(
+    printMock(
+      `Scenario routes registered: [\n${Object.keys(scenarios)
+        .map((key) => [
+          spacer,
+          `"${key}" - ${yellow(
+            `"${joinUrlFragments(serverAddress, (scenarios[key][0] || scenarios[key]).request)}"`
+          )}`,
+        ].join('')
+        )
+        .join(',\n')}\n  ]\n`
+    )
+  );
+}
+
+// Locales
+
+export function logLocaleAction({ locale, moduleName, action }) {
+  log(
+    printLocale(
+      `${orange(`"${locale}"`)} for module ${orange(`"${moduleName}"`)} has been ${action}`
+    )
+  );
+}
+
+export function logModuleLanguagePacksLoaded({ moduleName, languagePacks }) {
+  log(
+    printLocale(
+      `Loaded language packs for ${orange(`"${moduleName}"`)}: [ ${languagePacks
+        .map((langPack) => green(langPack))
+        .join(', ')} ]`
+    )
+  );
+}
+
+export function logLocaleModuleNamesBeingWatched(moduleNames) {
+  log(
+    printLocale(
+      `Watching language packs for modules: [ ${moduleNames
+        .map((name) => orange(JSON.stringify(name)))
+        .join(', ')} ]`
+    )
+  );
+}
+
+export function warnOnLocaleWatchError(err) {
+  warn(printLocale(`Language pack watcher error: ${err}`));
```

Comment:
```suggestion
export function warnOnLocaleWatchError(e) {
  warn(printLocale(`Language pack watcher error: ${e}`));
```

We're using just `e` everywhere else, though what you have here (`err`) is better, `error` would be even better

Reply:
went ahead with `error`